testsuite/cluster-test/src/health/mod.rs

```diff
@@ -87,15 +90,19 @@ impl HealthCheckRunner {
         }
     }
 
-    pub fn new_all(cluster: Cluster) -> Self {
+    pub fn new_all(cluster: Cluster, prometheus: Option<Prometheus>) -> Self {
+        let mut health_check: Vec<Box<dyn HealthCheck>> = vec![];
         let liveness_health_check = LivenessHealthCheck::new(&cluster);
-        Self::new(
-            cluster,
-            vec![
-                Box::new(CommitHistoryHealthCheck::new()),
-                Box::new(liveness_health_check),
-            ],
-        )
+        health_check.push(Box::new(CommitHistoryHealthCheck::new()));
+        health_check.push(Box::new(liveness_health_check));
+        match prometheus.as_ref() {
```

Comment:
use `if let` here if you only care about case when prometheus is set

Reply:
yeah, lint check noticed this format error, will update